config/locales/en/steps.yml

```diff
@@ -13,6 +13,8 @@ en:
       has_nino:
         edit:
           page_title: Enter your client’s NINO
+          heading: Do you have your client's National Insurance number?
```

Comment:
Needs to be: 'Enter your client's National Insurance number'
https://www.figma.com/file/thkvnDBbQbHlSqzt1TE3lk/Crime-Apply?node-id=2769%3A25139